src/microbe_stage/MicrobeCamera.cs

```diff
@@ -191,6 +194,11 @@ public bool Zoom(float delta, float value)
         return true;
     }
 
+    public override void _Process(float delta)
+    {
+        materialToUpdate.SetShaderParam("lightLevel", LightLevel);
```

Comment:
LightLevel should be made into a property that only applies this value to the shader when it changes.